phpstorm/CheckBuildUpdate.ps1

```diff
@@ -26,6 +26,15 @@ catch {
     }
 }
 
+# Jetbrains can be a little inconsistent in their naming of document pages.
+# So we verify the release specific page actually exist where we think it should.
+$release_url = "https://confluence.jetbrains.com/display/PhpStorm/PhpStorm+$newVersion+Release+Notes"
+$download_release = Invoke-WebRequest -Uri $release_url -UseBasicParsing
+# If not fallback to a documentation page that contains a list of all release note pages.
+if($download_release.RawContent -like '*Page Not Found*') {
```

Comment:
I'm not sure how long ago I picked this information up but I've been under the impression that Confluence did not return proper status codes in many cases. Checking now it would seem they have fixed that and now properly return a 404. I'll add this change now.